srcpkgs/qalculate-gtk/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'qalculate-gtk'
 pkgname=qalculate-gtk
```

Comment:
Add a comment above this to say it needs to be sync'd with `qalculate`, and vice versa